go18_test.go

```diff
@@ -176,6 +177,10 @@ func TestIssue617(t *testing.T) {
 			}
 		}()
 	}
+
+	// Give time for goroutines to terminate
+	time.Sleep(time.Millisecond * 50)
```

Comment:
This is a test flake waiting to happen. Can you put this into a retry loop with like a 50ms delay between attempts that fails after 1s or something?

Reply:
Will do!